src/time.cr

```diff
@@ -252,8 +252,13 @@ struct Time
   # :nodoc:
   DAYS_PER_4_YEARS = 365*4 + 1
 
-  # :nodoc:
-  UNIX_SECONDS = SECONDS_PER_DAY.to_i64 * (1969*365 + 1969/4 - 1969/100 + 1969/400)
+  # This constant is defined to be "1970-01-01 00:00:00 UTC" on all systems with respect to the system clock.
+  # Can be used to create a `Time::Span` that represents an unix epoch time duration.
+  #
+  # ```
+  # Time.utc_now - Time::UNIX_EPOCH
+  # ```
+  UNIX_EPOCH = utc(seconds: SECONDS_PER_DAY.to_i64 * (1969*365 + 1969/4 - 1969/100 + 1969/400), nanoseconds: 0)
```

Comment:
Why is this not just `Time.utc(1970, 1, 1)` ??

Reply:
I was using at first, but this doesn't work. @straight-shoota [explained it in 2.](https://github.com/crystal-lang/crystal/pull/6908#issuecomment-427022520)

Reply:
Alternatively, `zone_offset_at` could just be skipped if `location == UTC`. Then `Time.utc(1970, 1, 1)` wouldn't depend on `UNIX_EPOCH`.